dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/trackedentity/AbstractTrackedEntityInstanceService.java

```diff
@@ -200,61 +200,8 @@
     }
 
     @Override
-    @Transactional( readOnly = true )
```

Comment:
I see that the previous getTrackedEntityInstances2() method did not have the @Transactional (readOnly = true) declaration. Would it make sense to add it?

Reply:
I guess so, I will add it